.github/workflows/build_conda_pkg.yml

```diff
@@ -1,4 +1,4 @@
-name: Build Conda Package
+name: Build conda package
```

Comment:
Call me out if there's no point updating this lol, I just figured it'd be easier to read in the list